wezterm-gui/src/overlay/copy.rs

```diff
@@ -877,6 +878,61 @@ impl CopyRenderable {
         self.select_to_cursor_pos();
     }
 
+    // Move 1 vim WORD backward
+    fn move_backward_one_non_blank_word(&mut self) {
+        let y = if self.cursor.x == 0 && self.cursor.y > 0 {
+            self.cursor.x = usize::max_value();
+            self.cursor.y.saturating_sub(1)
+        } else {
+            self.cursor.y
+        };
+
+        let (top, lines) = self.delegate.get_lines(y..y + 1);
+        if let Some(line) = lines.get(0) {
+            self.cursor.y = top;
+            if self.cursor.x == usize::max_value() {
+                self.cursor.x = line.len().saturating_sub(1);
+            }
+            let s = line.columns_as_str(0..self.cursor.x.saturating_add(1));
+
+            let mut last_was_whitespace = false;
+            let mut passed_word = false;
+
+            for (idx, word) in s.split_word_bounds().rev().enumerate() {
+                let width = unicode_column_width(word, None);
+
+                if is_whitespace_word(word) {
+                    if passed_word {
+                        self.cursor.x = self.cursor.x.saturating_add(1);
+                        break;
+                    }
+                    self.cursor.x = self.cursor.x.saturating_sub(width);
+                    last_was_whitespace = true;
+                    continue;
+                }
+                last_was_whitespace = false;
+
+                if idx == 0 && width == 1 {
+                    // We were at the start of the initial word
+                    self.cursor.x = self.cursor.x.saturating_sub(width);
+                    continue;
+                }
+
+                self.cursor.x = self.cursor.x.saturating_sub(width);
+                passed_word = true;
+            }
+
+            if last_was_whitespace && self.cursor.y > 0 {
+                // The line begins with whitespace
+                self.cursor.x = usize::max_value();
+                self.cursor.y -= 1;
+                return self.move_backward_one_word();
```

Comment:
I think this should be `self.move_backward_one_non_blank_word()`
```suggestion
                return self.move_backward_one_non_blank_word();
```

Reply:
I think you also need to update call to `self.move_to_end_of_non_blank_word` in the definition of `move_to_end_of_non_blank_word` similar to the above mentioned change